### DISCUSSION TITLE
If I run code-server on Linux while the client is Windows or ChromeOS which keyboard shortcuts does code-server use?

### DISCUSSION THREAD

**Discussion by partev** [coder/code-server#7000] [Q&A]
code-server has different keyboard shortcuts depending on the OS:
e.g. https://code.visualstudio.com/shortcuts/keyboard-shortcuts-linux.pdf or https://code.visualstudio.com/shortcuts/keyboard-shortcuts-windows.pdf

are those shortcuts determined by the server OS or the client OS?

**Comment by code-asher** [comment]
I am fairly certain it uses the client OS.
